src/blocks/net.rs

```diff
@@ -423,6 +425,35 @@ impl ConfigBlock for Net {
             .with_default("{speed_down;K}{speed_up;K}")?;
         let format_alt = block_config.format_alt;
 
+        let tx_format = format.get("graph_up");
+        let rx_format = format.get("graph_down");
+        // prefers the full option
```

Comment:
That's not ideal. Right now I don't have a recommendation on how to properly implement this, but short config should not be ignored.